### PR TITLE
Add checksum for py-onnxmltools 1.11.1

### PR DESCRIPTION
**Pull request by iarspider**

### CODE REVIEW

**Comment by iarspider** [comment]
@spackbot fix style

**Review by tldahlgren — COMMENTED** [review]
Confirmed sha256.

File: var/spack/repos/builtin/packages/py-onnxmltools/package.py [hunk 1]
```diff
@@ -12,6 +12,13 @@ class PyOnnxmltools(PythonPackage):
     homepage = "https://github.com/onnx/onnxmltools"
     pypi = "onnxmltools/onnxmltools-1.10.0.tar.gz"
 
+    # Source tarball not available on PyPI
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-onnxmltools/package.py:15]: If this is the new normal we should update the `url` above and add a `url_for_version` so we don't have to specify the URL for all versions. Could also always use GitHub as the official source, even for older versions.

Comment (iarspider) [reply var/spack/repos/builtin/packages/py-onnxmltools/package.py:15]: I'd wait for next release: if they will not put source tarball on PyPI, then we can change the URL.

**Review by adamjstewart — APPROVED** [review]

**Comment by iarspider** [comment]
@spackbot run pipeline

**Pull request merged**